doc/classes/PhysicsServer2DExtension.xml

```diff
@@ -935,6 +948,13 @@
 			<description>
 			</description>
 		</method>
+		<method name="_space_step" qualifiers="virtual" is_experimental="true">
+			<return type="void" />
+			<param index="0" name="space" type="RID" />
+			<param index="1" name="delta" type="float" />
```

Comment:
Shouldn't this be of type `real_t`? (I have no clue, I just read about the precision=double compile option).
I find it weird that in `_PhysicsProcess(delta double)` you can get double but the physics only allow to pass float. Did I confuse something?

Reply:
These type infomation is generated.
There have not a type named `real_t` or `double` in GDScript, only have `float`.